common/src/test/scala/uk/ac/wellcome/sqs/SQSReaderTest.scala

```diff
@@ -0,0 +1,53 @@
+package uk.ac.wellcome.sqs
+
+import com.amazonaws.services.sqs.model.{Message, ReceiveMessageResult}
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{FunSpec, Matchers}
+import uk.ac.wellcome.models.aws.SQSConfig
+import uk.ac.wellcome.test.utils.SQSLocal
+
+import scala.collection.JavaConversions._
+import scala.concurrent.duration._
+
+class SQSReaderTest
+    extends FunSpec
+    with Matchers
+    with ScalaFutures
+    with IntegrationPatience
+    with SQSLocal {
+
+  it("should get messages from the SQS queue, limited by the maximum number of messages and return them") {
+    val sqsConfig =
+      SQSConfig("eu-west-1", idMinterQueueUrl, waitTime = 20 seconds, 2)
```

Comment:
Can we add a `maxMessages =` to this SQSConfig constructor please? Improves clarity and readability.